drivers/spi/Kconfig.stm32

```diff
@@ -32,5 +32,22 @@ config SPI_STM32_USE_HW_SS
 	help
 	  Use Slave Select pin instead of software Slave Select.
 
+config SPI_STM32F7_ERRATA_BUSY
+	bool
+	default y
+	depends on SOC_STM32F745XX || SOC_STM32F746XX || \
+		   SOC_STM32F750XX || SOC_STM32F756XX
+	help
+	  Handles erratum "BSY bit may stay high at the end of a data
+	  transfer in Slave mode".
+	  Seen in Errata Sheet 0290 §2.11.2
+
+if SPI_STM32F7_ERRATA_BUSY
+
+config SPI_STM32_BUSY_FLAG_WORKAROUND_TIMEOUT_US
+	int "timeout in us for the STM32 busy flag workaround"
```

Comment:
Timeout is now in ms (see WAIT_FOR header)

Reply:
Are you sure?

    @param timeout Timeout to wait for in microseconds, e.g.: 1000 (1ms)

Reply:
You're right of course.
My brain translated microseconds into "ms". That is a good signal for end of the day!
Sorry for the noise